### PR TITLE
Add comments in lxqtpanelapplication.h and correct comments in ilxqtpanelplugin.h

### PR DESCRIPTION
**Pull request by DrTobe**
Third try :-)

### CODE REVIEW

File: panel/lxqtpanelapplication.h [hunk 1]
```diff
@@ -39,37 +39,140 @@ namespace LXQt {
...
+     * \return the current number of LXQtPanel objects
+     */
     int count() { return mPanels.count(); }
+    
+    /*!
+     * \brief Checks if a given Plugin is running and has the
+     * ILXQtPanelPlugin::SingleInstance flag set. As Plugins are added to
+     * LXQtPanel instances, this method only iterates over these LXQtPanel
+     * instances and lets them check the conditions.
+     * \param pluginId TODO
```
Comment (palinek) [inline panel/lxqtpanelapplication.h]: This is the plugin identifier -> basename of .desktop file w/o the .desktop extension: e.g.  clock.desktop -> clock

**Comment by palinek** [comment]
After the TODO fix. GTM

**Comment by DrTobe** [comment]
Now, it should be ok.

**Comment by palinek** [comment]
OK. after squashing these 2 commits, GTM

**Comment by DrTobe** [comment]
I thought I would leave it like that because it is something different. On the other side, no program code is changed so there will never be a reason to undo these commits at all. You want me to squash those two?

**Comment by palinek** [comment]
Sorry.. thouhgt you have added another commit for the 'TODO fix' ... no, you don't need to squash these two.

GTM

**Comment by paulolieuthier** [comment]
Merged. Thanks.

**Pull request closed without merging**